code/modules/mob/living/carbon/human/husk.dm

```diff
@@ -0,0 +1,80 @@
+/datum/species/husk
+	name = "Husk"
+	name_plural = "Husks"
+	icobase = 'icons/mob/human_races/r_husk.dmi'
+	deform = 'icons/mob/human_races/r_husk.dmi'
+	total_health = 200
+	species_flags = NO_BREATHE|NO_SCAN|NO_BLOOD|NO_POISON|NO_PAIN|NO_CHEM_METABOLIZATION|NO_STAMINA|DETACHABLE_HEAD|HAS_UNDERWEAR|NO_HEALTH_HUD
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+	see_in_dark = 8
+	blood_color = "#110a0a"
+	hair_color = "#000000"
+	slowdown = 1.5
+	unarmed_type = /datum/unarmed_attack/husk
+	default_language_holder = /datum/language_holder/zombie
+	///Sounds made randomly by the zombie
+	var/list/sounds = list('sound/hallucinations/growl1.ogg','sound/hallucinations/growl2.ogg','sound/hallucinations/growl3.ogg','sound/hallucinations/veryfar_noise.ogg','sound/hallucinations/wail.ogg')
+	///Time before resurrecting if dead
+	var/revive_time = 1 MINUTES
+	///How much burn and burn damage can you heal every Life tick (half a sec)
+	var/heal_rate = 5
+	///How much zombium are transferred per hit. Set to zero to remove transmission
+	var/zombium_per_hit = 5
+
+/datum/species/husk/on_species_gain(mob/living/carbon/human/H, datum/species/old_species)
+	H.language_holder = new default_language_holder()
+	H.status_flags |= CANNOT_HOLD //Failsafe if something manage to skip the attack_hand check
+	H.setOxyLoss(0)
+	H.setToxLoss(0)
+	H.setCloneLoss(0)
+	H.dropItemToGround(H.r_hand)
+	H.dropItemToGround(H.l_hand)
+	//remove larva
+	var/obj/item/alien_embryo/alien_embryo = locate() in src
+	if(alien_embryo)
+		qdel(alien_embryo)
+		return
+
+/datum/species/husk/post_species_loss(mob/living/carbon/human/H)
+	H.status_flags &= ~CANNOT_HOLD
```

Comment:
I dislike this status flag for two reasons:
A) it should just be a trait
B) A better option (imo) is to just equip nodrop equipment to the zombies hands so we can have something like gun zombies potentially

Reply:
This is just a fail safe. I can probably remove and it will not change anything